org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/ExtensionCompiler.scala

```diff
@@ -359,3 +381,38 @@ object ExtensionCompiler extends AnyRef with HasLogger {
   }
 
 }
+
+/**
+ * Generator for the creator classes of the IDE extensions. Do not forget to
+ * specify the `saveLocation` before you run this App. It needs to point to
+ * something like `/path/to/scala-ide/org.scala-ide.sdt.core/src`. All the
+ * generated sources are written together with their package hierarchy into this
+ * directory.
+ */
+object ExtensionGenerator extends App {
+
+  private val saveLocation = "/home/antoras/dev/scala/scala-ide/org.scala-ide.sdt.core/src"
```

Comment:
Yeah, but as I said: it depends on `sdt.core` and the whole eclipse. How do you ran the `App`? What classpath did you pass?